src/Symfony/Component/Console/Command/Command.php

```diff
@@ -312,22 +313,6 @@ public function complete(CompletionInput $input, CompletionSuggestions $suggesti
      */
     public function setCode(callable $code): static
     {
-        if ($code instanceof \Closure) {
-            $r = new \ReflectionFunction($code);
-            if (null === $r->getClosureThis()) {
-                set_error_handler(static function () {});
-                try {
-                    if ($c = \Closure::bind($code, $this)) {
-                        $code = $c;
-                    }
-                } finally {
-                    restore_error_handler();
-                }
-            }
-        } else {
-            $code = $code(...);
-        }
```

Comment:
Moved to `InvokableCommand` which also supports handling static anonymous functions